public/app/features/plugins/admin/api.ts

```diff
@@ -37,6 +37,8 @@ export async function getPluginDetails(id: string): Promise<CatalogPluginDetails
     iam: remote?.json?.iam,
     lastCommitDate: remote?.lastCommitDate,
     changelog: remote?.changelog || localChangelog,
+    licenseUrl: remote?.licenseUrl,
```

Comment:
since those fields are optional on the gcom side you should handle the case where they are not present in the API response.

Reply:
I have tested and it should be fine. in case the property does not exist it will be just undefined.